src/Roslyn.Suppressions/Roslyn.Suppressions.UnitTests/TestHelpers.cs

```diff
@@ -56,4 +59,30 @@ internal static class TestHelper
                 null
                 );
     }
+
+    //This is to make sure normalising the keys done correctly with culture invariant
+    //Lower case of SETTINGSKEY in Turkish is not settingskey but settıngskey
+    //https://en.wikipedia.org/wiki/Dotted_and_dotless_I 
+    public class TemporaryCultureSwitch : IDisposable
+    {
+        private readonly CultureInfo _originalCulture;
+        private readonly CultureInfo _originalUICulture;
+
+        public TemporaryCultureSwitch(CultureInfo cultureInfo)
```

Comment:
FYI we have other test-only disposable types that make a temporary change. By convention they're named `Scope` rather than `Switch` e.g. `AssertIgnoreScope`.